### PR TITLE
[ADD] pos_product_quick_info: Add new module to display product without add in to card

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi. tested on runboat. Looks ok. 
Functional question : how do you use this popup ? I see the interest, but It looks hugly and not responsive at all. (close button is hidden if the screen is not big).

**Comment by ivs-cetmix** [comment]
> Hi. tested on runboat. Looks ok.
> Functional question : how do you use this popup ? I see the interest, but It looks hugly and not responsive at all. (close button is hidden if the screen is not big).

Thank you for the feedback! The idea behind this module is to be able to check product info without adding it to cart.
Regarding "Close" button what do you think if popup will be closed when clicked on popup itself?

**Review by legalsylvain — APPROVED** [review]
Code review and test. Lgtm. Thanks !

**Comment by legalsylvain** [comment]
> Thank you for the feedback! The idea behind this module is to be able to check product info without adding it to cart.
Regarding "Close" button what do you think if popup will be closed when clicked on popup itself?

Well. Just tested on smartphone and the display is correct. In fact, it seems bad displayed only on médium screen. Should be fixed in odoo or in another dedicated module.

Thanks for your contribution.

**Comment by ivs-cetmix** [comment]
@OCA/pos-maintainers would appreciate someones review

File: pos_product_quick_info/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "PoS Product Quick Info",
+    "version": "16.0.1.0.0",
+    "summary": "Display product info by one click in Point of Sale",
+    "author": "Cetmix,Odoo Community Association (OCA)",
```
Comment (ivantodorovich) [inline pos_product_quick_info/__manifest__.py]:
```suggestion
    "author": "Cetmix, Odoo Community Association (OCA)",
```

**Review by ivantodorovich — APPROVED** [review]

File: pos_product_quick_info/__manifest__.py (hunk 1 above)
Comment (ivs-cetmix) [reply pos_product_quick_info/__manifest__.py]: Thank you very much!

**Review by ivs-cetmix — APPROVED** [review]
Functional ok!

**Review by ivs-cetmix — APPROVED** [review]

**Review by PhilDL — APPROVED** [review]

**Pull request merged**